macros/src/standard/nep141.rs

```diff
@@ -42,31 +42,35 @@ pub fn expand(meta: Nep141Meta) -> Result<TokenStream, darling::Error> {
     });
 
     Ok(quote! {
-        impl #imp near_contract_tools::standard::nep141::Nep141Controller for #ident #ty #wher {
-            fn root(&self) -> near_contract_tools::slot::Slot<()> {
-                near_contract_tools::slot::Slot::root(#storage_key)
+        use ::near_contract_tools::standard::nep141::Nep141Controller;
+
+        impl #imp ::near_contract_tools::standard::nep141::Nep141Controller for #ident #ty #wher {
+            fn root(&self) -> ::near_contract_tools::slot::Slot<()> {
+                ::near_contract_tools::slot::Slot::root(#storage_key)
             }
         }
 
-        #[near_sdk::near_bindgen]
-        impl #imp near_contract_tools::standard::nep141::Nep141 for #ident #ty #wher {
+        use ::near_contract_tools::standard::nep141::Nep141;
+
+        #[::near_sdk::near_bindgen]
+        impl #imp ::near_contract_tools::standard::nep141::Nep141 for #ident #ty #wher {
             #[payable]
             fn ft_transfer(
                 &mut self,
-                receiver_id: near_sdk::AccountId,
-                amount: near_sdk::json_types::U128,
+                receiver_id: ::near_sdk::AccountId,
```

Comment:
@encody What do these changes mean (adding the `::` prefixes)?

Reply:
> Beginning with the 2018 Edition, paths starting with :: resolve from crates in the [extern prelude](https://doc.rust-lang.org/reference/names/preludes.html#extern-prelude). That is, they must be followed by the name of a crate.

https://doc.rust-lang.org/reference/paths.html

Not all of the macros use this syntax yet, but they should. Just haven't gotten around to changing it yet.